tests/installation/bootloader_s390.pm

```diff
@@ -89,7 +89,8 @@ sub prepare_parmfile {
         $params .= " info=" . create_infofile("install: $instsrc");
     }
     else {
-        $params .= " install=" . $instsrc . $repo . " ";
+        #$params .= " install=" . $instsrc . $repo . " ";
```

Comment:
we don't need commented code now that we have a PR opened.

Reply:
Removed these commented code.